chia_py_streamable_macro/src/lib.rs

```diff
@@ -55,8 +55,8 @@ pub fn py_streamable_macro(input: TokenStream) -> TokenStream {
     };
 
     let mut py_protocol = quote! {
-        #[pyproto]
-        impl pyo3::class::basic::PyObjectProtocol for #ident {
```

Comment:
this way of defining special members was deprecated and removed in the new version